sqlglot/dialects/clickhouse.py

```diff
@@ -33,11 +33,20 @@ class Tokenizer(tokens.Tokenizer):
             "FINAL": TokenType.FINAL,
             "FLOAT32": TokenType.FLOAT,
             "FLOAT64": TokenType.DOUBLE,
+            "INT8": TokenType.TINYINT,
+            "UINT8": TokenType.UTINYINT,
             "INT16": TokenType.SMALLINT,
+            "UINT16": TokenType.USMALLINT,
             "INT32": TokenType.INT,
+            "UINT32": TokenType.UINT,
             "INT64": TokenType.BIGINT,
-            "INT8": TokenType.TINYINT,
+            "UINT64": TokenType.UBIGINT,
+            "INT128": TokenType.INT128,
+            "UINT128": TokenType.UINT128,
+            "INT256": TokenType.INT256,
+            "UINT256": TokenType.UINT256,
             "TUPLE": TokenType.STRUCT,
+            # "MAP": TokenType.STRUCT,
```

Comment:
Why is this here? I think we already handle this correctly:
```python
>>> sqlglot.transpile("a::map", "clickhouse")
['CAST(a AS Map)']
```

Reply:
You do, I meant to remove that thank you!